haiku-apps/catkeyseditor/catkeyseditor-0.1.1.recipe

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="https://github.com/puckipedia/BeLocalized"
 COPYRIGHT="2014-2015 Puck Meerburg"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
Can anyone confirm that it is OK to bump catkeyseditor-0.1.1.recipe to the new srcGitRev?
I'm sure @dash102 would like to know if she can bump REVISION for that recipe (and have her PR merged.)
If we wait too long then other students will have conflicts on their PRs if they fix the **`$(HOMEPAGE)`** errors in one of the recipes already being fixed here.
Thanks!